pennylane/ops/__init__.py

```diff
@@ -29,16 +29,22 @@
 from .cv import ops as _cv__ops__
 from .cv import obs as _cv__obs__
 
-from .qubit import __all__ as _qubit__all__
-from .qubit import ops as _qubit__ops__
-from .qubit import obs as _qubit__obs__
-
 from .qutrit import __all__ as _qutrit__all__
 from .qutrit import ops as _qutrit__ops__
 from .qutrit import obs as _qutrit__obs__
 
 from .channel import __all__ as _channel__ops__
 
+from .qubit import __all__ as _qubit__all__
+from .qubit import ops as _qubit__ops__
+from .qubit import obs as _qubit__obs__
+
+# we would like these to just live in .qubit, but can't because of circular imports
+from .op_math import controlled_qubit_ops as _controlled_qubit__ops__
+
+_qubit__ops__ = _qubit__ops__ | _controlled_qubit__ops__
+_qubit__all__ = _qubit__all__ + list(_controlled_qubit__ops__)
+
```

Comment:
Why not just import `ControlledQubitUnitary` in the top `__init__.py` file instead?